docs_user/assemblies/assembly_configuring-isolated-networks.adoc

```diff
@@ -6,29 +6,18 @@ ifdef::context[:parent-context: {context}]
 
 = Configuring isolated networks
 
-At this point, you should have a good idea about VLAN and IPAM configuration
-you would like to replicate in the new environment.
+Before you begin replicating your existing VLAN and IPAM configuration in the {rhos_long} environment, you must have the following IP address allocations for the new control plane services:
```

Comment:
@booxter Does the sentence make sense as to what the customer is going to configure in this assembly?

"Before you begin replicating your existing VLAN and IPAM configuration in the {rhos_long} environment, you must have the following IP address allocations for the new control plane services:"